docker/shell.entrypoint.sh

```diff
@@ -1,24 +1,5 @@
 #!/bin/bash
 set -e
 
```

Comment:
If you took this entrypoint out of the docker compose and docker file, then I think we don't need it at all.

Reply:
correct!